ci-operator/config/openshift/assisted-test-infra/openshift-assisted-test-infra-release-ocm-2.5.yaml

```diff
@@ -80,17 +80,6 @@ tests:
         DISK_ENCRYPTION_ROLES=all
         DISK_ENCRYPTION_MODE=tpmv2
     workflow: assisted-baremetal
-- as: e2e-metal-assisted-tpmv2-periodic
```

Comment:
also the presubmit ones ^
here and elsewhere

Reply:
Why?

Reply:
I thought of removing it because it uses the REST API to do the installation, meaning we don't really validate the regular interface for setting the relevant values
For this specific job maybe we can leave it (and replace afterwards with a new job) but for the lso/cnv/ocs/odf ones we should definitely remove the presubmit ones because we do not support them at all